sdx-encrypted-submitter.go

```diff
@@ -93,7 +122,7 @@ func getMessage(filePath string) ([]byte, error) {
 	return msgBody, nil
 }
 
-func sendToRabbit(url string, exchange string, routingKey string, msgBody []byte) error {
+func sendToRabbit(url string, exchange string, routingKey string, txID string, msgBody []byte) error {
```

Comment:
Idiomatically, if you've got a list of params that use the same type you can declare the type just once:
```go
sendToRabbit(url, exchange, routingKey, txID string, msgBody []byte) error { ... }
```

Reply:
Done , but I could argue the toss on this one on readability . I know  the zen of python is not admissible evidence M'lord but Explicit is better than implicit ?